core/data.go

```diff
@@ -40,6 +40,17 @@ type Blob struct {
 	Data          []byte
 }
 
+type BlobAuthHeader struct {
+	// Commitments
+	BlobCommitments `json:"commitments"`
```

Comment:
can you comment on what this means and why we need this field?

Reply:
Added a comment to the `BlobAuthHeader`.